src/config.rs

```diff
@@ -200,30 +231,48 @@ impl ConfigBuilder {
     }
 
     /// Set time format string to use rfc2822.
+    #[cfg(not(feature = "chrono"))]
     pub fn set_time_format_rfc2822(&mut self) -> &mut ConfigBuilder {
```

Comment:
I would like to see this function being supported, when chrono is enabled as well. See comment for `write_time` below.